swagger_spec_validator/__about__.py

```diff
@@ -12,7 +12,8 @@
 __summary__ = "Validation of Swagger specifications"
 __uri__ = "http://github.com/Yelp/swagger_spec_validator"
 
-__version__ = "2.4.3"
+# version from original repos was 2.4.3
+__version__ = "2.5.1"
```

Comment:
What's the reasoning behind using 2.5.1? Maybe include that in the comment?

Reply:
Not sure if I really have much reasoning. Besides incrementing the minor number and resetting the patch number. The major should stay 2.   

Is that what your thinking should be in there?

Reply:
With that line of reasoning, I would have thought it would be 2.5.0. I'm not sure if anything bad would happen if we used a version number that's the same as a future version number released by the parent repo. If so, it might be safest to do something like 2.4.3.1 (though I know that doesn't really adhere to the usual rules of semantic versioning). I don't suppose it matters much as long as we have a test that will fail if we somehow wind up with the wrong version of this installed.

Second thing, though: is this file actually what gets used by pip to determine version info for what it installs?

Reply:
Changed the version, I'm indifferent what it should be.

This package deployment via git is new to me. I tested it and it works magically. Per version I don't think we can specify version if we are providing a pypy somewhere. I just pip install it directly from git. Git doesn't really know any idea of versions. We could force it using tags I guess. 

@23jodys if we are installing directly from git and not from a pypi do we need to worry about versions?

Reply:
What version number does pip say is installed after installing from the git repo?

Reply:
2.5.1. 

Which is what is in setup.py after it imports it from about.

Reply:
👍 sweet